compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/JvmValueGen.java

```diff
@@ -429,6 +436,32 @@ private void createTypeDescConstructor(ClassWriter cw) {
         mv.visitEnd();
     }
 
+    private void createTypeDescConstructorWithAnnotations(ClassWriter cw, String name) {
+
```

Comment:
Lets remove these new lines in the method. Better to remove with separate PR.

Reply:
Updated(https://github.com/ballerina-platform/ballerina-lang/pull/39349)